crates/compiler/problem/src/lib.rs

```diff
@@ -6,6 +6,10 @@ pub mod can;
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum Severity {
+    /// This should stop compilation in all cases.
+    /// Due to delayed loading of ingested files, this is wanted behaviour over a runtime error.
+    Fatal,
+
```

Comment:
I think we could make this a runtime error instead - I think a goal is that we eventually want no runtime-blocking errors, as much as possible. Is there an inherit reason missing files can't be a runtime error?

Reply:
Brought up on zulip, keeping fatal as suggested by Richard.